mmv1/products/cloudrunv2/api.yaml

```diff
@@ -0,0 +1,717 @@
+              name: "type"
+              output: true
+              description: |-
+                type is used to communicate the status of the reconciliation process. See also: https://github.com/knative/serving/blob/main/docs/spec/errors.md#error-conditions-and-reporting Types common to all resources include: * "Ready": True when the Resource is ready.
+            - !ruby/object:Api::Type::String
+              name: "status"
+              output: true
+              description: |-
+                State of the condition.
+            - !ruby/object:Api::Type::String
+              name: "message"
+              output: true
+              description: |-
+                Human readable message indicating details about the current status.
+            - !ruby/object:Api::Type::Time
+              name: "lastTransitionTime"
+              output: true
+              description: |-
+                Last time the condition transitioned from one status to another.
+
+                A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+            - !ruby/object:Api::Type::String
+              name: "severity"
+              output: true
+              description: |-
+                How to interpret failures of this condition, one of Error, Warning, Info
+            - !ruby/object:Api::Type::String
+              name: "reason"
+              output: true
+              description: |-
+                A common (service-level) reason for this condition.
+            - !ruby/object:Api::Type::String
+              name: "revisionReason"
+              output: true
+              description: |-
+                A reason for the revision condition.
+            - !ruby/object:Api::Type::String
+              name: "executionReason"
+              output: true
+              description: |-
+                A reason for the execution condition.
+      - !ruby/object:Api::Type::String
+        name: "latestReadyRevision"
+        output: true
+        description: |
+          Name of the latest revision that is serving traffic. See comments in reconciling for additional information on reconciliation process in Cloud Run.
+      - !ruby/object:Api::Type::String
+        name: "latestCreatedRevision"
+        output: true
+        description: |
+          Name of the last created revision. See comments in reconciling for additional information on reconciliation process in Cloud Run.
+      - !ruby/object:Api::Type::Array
+        name: "trafficStatuses"
+        description: |-
+          Detailed status information for corresponding traffic targets. See comments in reconciling for additional information on reconciliation process in Cloud Run.
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Enum
```

Comment:
It looks like you've used `!ruby/object:Api::Type::Enum` for enum types that are provided by the user, which makes sense to me, but here since it is output only, would a `String` be more appropriate, like you did for the `conditions` object above?

Reply:
fixed